Add info on how to change values on G-010G-P (#48)
hack-gpon/hack-gpon.github.io: _ont/ont-nokia-g-010g-p.md

| System      |          |
| HSGMII      | No       |
| Optics      | SC/APC   |
| IP address  | 192.168.100.1 |
| Web Gui     |          |
| Telnet      | ✅ user `root`, password `huigu309` |
| Form Factor | ONT      |

{% include image.html file="g-010g-p.jpg"  alt="G-010G-P" caption="Nokia G-010G-P" %}

> To get a "full" linux shell just type:

Reviewer: ```suggestion
To enter in a complete linux shell just type:
```

```sh
#ONT>system
#ONT/system>shell

> # 

Reviewer: ```suggestion
```
```

```

# Useful Commands

## Committing changes to the OMCI MIB tables for GPON operation
```sh
#ONT>system
#ONT/system>mib
#ONT/system/mib>reset
```

## Getting/Setting the ONT's S/N
```sh
#ONT>system
#ONT/system>misc

> #ONT/system/misc>eqsn set "GNXS05546fb0"

Reviewer: move get before set

---ATECMDRESULT--- OK
#ONT/system/misc>eqsn get
eqsn: GNXS05546fb0
---ATECMDRESULT--- OK
```

## Getting/Setting the ONT's Vendor
```sh
#ONT>system
#ONT/system>misc

> #ONT/system/misc>eqsnvend set "ALCL"

Reviewer: move get before set

---ATECMDRESULT--- OK
#ONT/system/misc>eqsnvend get
```

## Getting/Setting the ONT's Equipment ID
```sh
#ONT>system
#ONT/system>misc
#ONT/system/misc>eqid set "FT-G2110C-2.5G"
---ATECMDRESULT--- OK

> #ONT/system/misc>eqid get

Reviewer: move get before set

eqid: FT-G2110C-2.5G
hex_eqid: 0x46542d4732313130432d322e3547000000000000
---ATECMDRESULT--- OK
```

## Getting/Setting the ONT's Hardware Version
```sh
#ONT>system
#ONT/system>misc
#ONT/system/misc>eqvid get
eqvid: 3FE45458ABAA06
hex_eqvid: 0x3346453435343538414241413036
---ATECMDRESULT--- OK
#ONT/system/misc>eqvid set "G2110CE2V1D0"
```

## Querying a particular OMCI ME
```sh
#ONT>system
#ONT/system>mib
#ONT/system/mib>show 256
Table Ontg, Ont-g, total 1 instances

EntityID                  = 0x0000
VID                       = "GNXS"
Version                   = 47 32 31 31 30 43 45 32 56 31 44 30 00 00
SerialNum                 = 47 4e 58 53 05 54 6f b0
TraffMgtOpt               = 0
AtmCCOpt                  = 0
BatteryBack               = 1
AdminState                = 0
OpState                   = 0
OnuSurvivalTime           = 0
Loid                      = ""
Password                  = ""
AuthState                 = 0
OntState                  = 1
```

> # Setting the ONT's Software Version

Reviewer: ```suggestion
## Setting the ONT's Software Version
```
